resources/ssm_parameter_store.rb

```diff
@@ -6,6 +6,13 @@
 property :with_decryption,             [true, false], default: false
 property :allowed_pattern,             String
 property :return_key,                  String
+property :names,                       [String, Array], required: true
```

Comment:
Was this previously required but not marked required?

Reply:
No.  names is new.  There used to be a name field but that was removed because it wasn't needed.